cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -2838,6 +2839,7 @@ display.page.search.mode = fast
 request.ui_dispatch_app = cyences_app_for_splunk
 request.ui_dispatch_view = search
 search = `cs_o365` sourcetype="o365:management:activity" _index_earliest=-61m@m _index_latest=-1m@m Workload IN (OneDrive, SharePoint) Operation IN ("AnonymousLinkUsed","SharingLinkUsed") NOT TargetUserOrGroupType IN ("Member", "SharePointGroup") \
+|`cs_o365_excluding_microsoft_teams_filter` \
```

Comment:
update the macro name: cs_o365_link_accessed_by_external_user_internal_filter
and add it to the macros.conf and remove the cs_o365_excluding_microsoft_teams_filter